cmd/thanos/query.go

```diff
@@ -28,18 +31,24 @@ import (
 	"github.com/prometheus/prometheus/promql"
 	"github.com/prometheus/tsdb/labels"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 // registerQuery registers a query command.
```

Comment:
BTW what happened to GitHub -  coloring for golang sucks ... now there is some red mark on go comments??

Reply:
It's doing it for me too, I've no idea what is going on there.